frontend/src/dto/ApplyClientNumberDto.ts

```diff
@@ -29,6 +29,10 @@ export interface FormDataDto {
     goodStandingInd: string;
     birthdate: string;
     address: Address;
+    middleName?: string;
```

Comment:
I would suggest also adding first and last name fields just to make it easier for the backend.

Reply:
Hey, what I did at the same time is watch the three field names and use them to update the `businessName` field.
So maybe I should actually just remove this `middleName` from the DTO.
What do you think? Or maybe I can keep it and the backend can just ignore it...

Reply:
Ok, I found out the backend actually expects to have the firstName and lastName in the businessInformation as well, so I will do it.